Diff_Results.txt

```diff
@@ -0,0 +1,13 @@
+[Inputs]
```

Comment:
This file seems to be in the base level of RAVEN, but should it be?

Reply:
I am okay based on Dylan's description of the diffing script's location, but I still believe this one is probably not in the right location.

Reply:
I removed this file and in the process of testing the best location on putting the outputs. I am thinking executing the diff through alias and put the output in the same folder but not sure if that will work for a custom diff.